Cargo.toml

```
@@ -64,6 +64,12 @@ hostname = "^0.3"
 [target.'cfg(all(target_arch = "wasm32", target_os = "unknown"))'.dependencies]
 web-sys = { version = "^0.3", features = ["Window", "Location"] }
 
+[target."cfg(not(target_os = \"wasi\"))".dependencies]
```

Is there a reason you did not use single ticks here, like in the other targets/?
```suggestion
[target.'cfg(not(target_os = "wasi"))'.dependencies]
```